.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+exclude: ^((examples|other)\/.*\/.*\.(py|poscar|csv))
+repos:
+  - repo: https://github.com/mrakitin/exabyte-io-pre-commit-hooks
+    rev: 09cf1c797140872602bae04b950e3afb5c38a9ee  # mrakitin/SOF-6690-hooks-isort-mypy branch
+    hooks:
+      - id: ruff
+      - id: black
+      - id: isort
+      - id: mypy
+        exclude: ^examples\/config\.py
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
```

Comment:
I'm personally OK with having hooks such as these basic ones (no config options) separate from our repo as is done here. Thoughts @timurbazhirov ?

Reply:
Thanks! Yes, those looked pretty simple, and the corresponding .py code lives in the same repo, so it's a bit too much work to port it to this org.

Reply:
I think it's better to build a unified setup for ourselves in one place and reuse it for all the repos from there. If we need to, we can include lines 11-16 in https://github.com/Exabyte-io/pre-commit-hooks

Reply:
https://github.com/Exabyte-io/pre-commit-hooks/pull/2 is to address the request.